### DISCUSSION TITLE
Upload folder - lots of folders and files in there

### DISCUSSION THREAD

**Discussion by Str1atum** [immich-app/immich#12617] [General]
Hi there!

I checked my filestructure of the immich folder and noticed that there are numerous folders and files withing the "upload" folder. I never manually placed anything there, I just used the web upload feature for everything and have a storage template activated.
Can I just wipe the upload folder - or for what purpose are the files in there?
There are mostly .mov files in there and most are partly corrupted.

Str1atum

<img width="1339" alt="Bildschirmfoto 2024-09-12 um 21 51 18" src="https://github.com/user-attachments/assets/f69e05c1-7573-4c39-8356-a0d13cb4a47b">

**Comment by mmomjian** [comment]
Those are folders, not files. This is the structure for the uploaded files, they get moved to the library/ folder with the storage template. I would recommend leaving them there, though the empty ones can be deleted (if you know what you are doing, and have good backups)
